bytecode/instr.py

```diff
@@ -268,11 +269,17 @@ def lineno(self):
     def lineno(self, lineno):
         self._set(self._name, self._arg, lineno)
 
+    @property
+    def stack_effect(self):
+        # All opcodes whose arguments are not represented by integers have
+        # a stack_effect indepent of their argument.
+        arg = (self._arg if isinstance(self._arg, int) else
+               0 if self._opcode >= _opcode.HAVE_ARGUMENT else None)
```

Comment:
I dislike isinstance(self._arg, int): for generic instruction (Instr class), _arg can have multiple types, not only int:
http://bytecode.readthedocs.io/en/latest/api.html#Instr

It seems like this method better suits for ConcreteInstr. What do you think?

In ConcreteInstr, the "has argument?" check is written "self._arg is not UNSET".

Note: I'm using UNSET because for Instr, _arg can be None, really the "value" None, Like LOAD_CONST(None).